### PR TITLE
Update gunicorn.rst for Gunicorn 20

### LINKED ISSUE #222: Updates for Gunicorn 20
In versions of Gunicorn prior to 20, use of `egg:gunicorn#main` was
deprecated when used with external script runners and the Gunicorn
documentation suggested use of the Gunicorn CLI directly.

As requests came into the Gunicorn issue tracker for the ability to advanced
Paste features like specifying globals for interpolation in the config
file and using alternate `server` blocks, it became clear that adding
these features would significantly inflate the surface of the Gunicorn
CLI. Furthermore, it was potentially confusing that using
the Gunicorn CLI while specifying another server in the `server` block
would still use Gunicorn as the server.

For version 20, it was decided that it is best to let the tools of the
Paste ecosystem manage invoking the server. It is no longer deprecated
to specify `egg:gunicorn#main` as the server and invoke Gunicorn with
`pserve`. Gunicorn is fully supported as a Paste Deploy server factory.
Additionally, the Gunicorn CLI fully supports reading a Paste Deploy app
factory from a `.ini` file. However, Gunicorn is _not_ a Paste Script runner.

I suggest removing the "short story" from the gunicorn.rst. It is better to
suggest that users change their `.ini` server block to use Gunicorn but to
keep the invocation of `pserve` as before.

### PR DESCRIPTION
**Pull request by tilgovi**
Close #222.

### CODE REVIEW

File: docs/deployment/gunicorn.rst [hunk 1]
```diff
@@ -35,16 +35,14 @@ accesslog lines to stderr and use the debug loglevel.
...
     accesslog = -
     loglevel = debug
 
 For all configuration options that may be used, have a look at the `available
 settings <http://docs.gunicorn.org/en/stable/settings.html>`_.
 
 Keep in mind that settings defined within a gunicorn configuration file or
-command line arguments given to gunicorn take precedence over the settings
-established within the INI file.
+take precedence over the settings established within the INI file.
```
Comment (stevepiercy) [inline docs/deployment/gunicorn.rst]: This sentence does not make sense, maybe remove the "or" from line 44?

File: docs/deployment/gunicorn.rst [hunk 2]
```diff
@@ -35,16 +35,14 @@ accesslog lines to stderr and use the debug loglevel.
...
     workers = 4
     preload = true
-    reload = true
     accesslog = -
     loglevel = debug
 
 For all configuration options that may be used, have a look at the `available
 settings <http://docs.gunicorn.org/en/stable/settings.html>`_.
 
 Keep in mind that settings defined within a gunicorn configuration file or
```
Comment (tilgovi) [inline docs/deployment/gunicorn.rst]:
```suggestion
Keep in mind that settings defined within a gunicorn configuration file
```

File: docs/deployment/gunicorn.rst (hunk 1 above)
Comment (tilgovi) [reply docs/deployment/gunicorn.rst]: Thank you!

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**